### PR TITLE
CMM Expansion

### PR DESCRIPTION
**Pull request by rye-rice**
## About The Pull Request
Some content to expand the CMM content in the game. It has been lacking, and might still be, but there is more to work with for mappers now.

image

NOTE: This does not touch any maps, this is more for mappers to mess with.

CM-15, Autmatic shotgun thats a reskin of the bulldog

image

CM-5, smg that uses 9mm
image

GAL, DMR that uses 308
image

CMM Patroller hardsuit, functionally similar to the security hardsuit
image

Sound effects ripped off [TGMC](https://github.com/tgstation/TerraGov-Marine-Corps) as usual

- [x] I affirm that I have tested all of my proposed changes and that any issues found during tested have been addressed.

## Why It's Good For The Game

More content for mappers, admins, and players for a faction lacking in content.

## Changelog

:cl:
add: CMM weapons CM-GAL-S and CM-5
add: Minutemen now have their own backpack and satchel along with hardsuit
add: Most minutemen sprites have been touched up in general
add: The Contender sprite is updated
tweak: Winchester 308 FMJ is now simply .308
balance: EBR no longer has burst fire
fix: Fixes the EBR
/:cl:

### CODE REVIEW

**Comment by genessee-forgot-his-password-again** [comment]
immensely based

File: code/game/objects/items/storage/belt.dm [hunk 1]
```diff
@@ -501,6 +501,14 @@
 	icon_state = "assaultbelt"
 	item_state = "security"
 
+/obj/item/storage/belt/military/assault/minutemen/PopulateContents()
+	new /obj/item/ammo_box/magazine/p16(src)
```
Comment (Fikou) [inline code/game/objects/items/storage/belt.dm]: use a loop

File: code/modules/projectiles/boxes_magazines/external/rifle.dm [hunk 2]
```diff
@@ -81,12 +81,37 @@
...
 	max_ammo = 10
 
+/obj/item/ammo_box/magazine/ebr/update_icon()
+	..()
+	var/hasammo
+	if(ammo_count())
+		hasammo = 1
+	else
+		hasammo = 0
+	icon_state = "ebr_mag-[hasammo]"
```
Comment (Fikou) [inline code/modules/projectiles/boxes_magazines/external/rifle.dm]:
```suggestion
	icon_state = "ebr_mag-[!!ammo_count()]"
```

File: code/modules/projectiles/boxes_magazines/external/rifle.dm [hunk 3]
```diff
@@ -81,12 +81,37 @@
...
+	name = "\improper M514 EBR Magazine (.308)"
 	icon_state = "ebr_mag"
-	ammo_type = /obj/item/ammo_casing/win308
-	caliber = ".308 Winchester FMJ"
+	ammo_type = /obj/item/ammo_casing/a308
+	caliber = ".308"
 	max_ammo = 10
 
+/obj/item/ammo_box/magazine/ebr/update_icon()
+	..()
```
Comment (Fikou) [inline code/modules/projectiles/boxes_magazines/external/rifle.dm]: you sure you dont wanna call parent?
also both of those reviews for the other mag too

**Review by Fikou — DISMISSED** [review]

**Comment by rye-rice** [comment]
Sorry for the delay, i'll get to this in a bit

**Comment by rye-rice** [comment]
image

**Comment by Latentish** [comment]
marg approved margtest mergery

**Pull request merged**